src/gtimelog/tests/test_main.py

```diff
@@ -47,7 +47,7 @@ def test_prepare_message_unicode(self):
             sender='Ünicødę Name <test@example.com>',
             recipient='Anöther nąme <activity@example.com>',
             subject='Report for Mr. ☃',
-            body='These are the activites done by Mr. ☃:\n...\n',
+            body='These are the activities done by Mr. ☃:\n...\n',
```

Comment:
I'm afraid this changes the base64-encoded message body to also change, which causes the test to fail.